examples/hashing/src/main.sw

```diff
@@ -2,10 +2,10 @@ script;
 
 use std::{
     chain::log_b256,
-    constants::ZERO,
     core::num::*,
     hash::{HashMethod, hash_pair, hash_u64, hash_value, sha256}
 };
+use core::num::*;
```

Comment:
Hmm @sezna would it be a good idea to implicitly import `core::num::*` in the predule too?